src/vrl/selectielijst/management/commands/load_data_from_excel.py

```diff
@@ -0,0 +1,137 @@
+import re
+from dateutil.relativedelta import relativedelta
+from xlrd import open_workbook
+
+from django.core.management.base import BaseCommand
+
+from vrl.selectielijst.models import ProcesType, Resultaat
+from vrl.selectielijst.constants import ArchiefNominaties, Procestermijnen
+
+
+# python src/manage.pload_data_from_excel /home/anna/Downloads/20170704-ontwerpselectielijst_opsomming_excel_definitief_versie2_0.xls
+# curl -X POST http://127.0.0.1:8081/api/v1/resultaten -H "Content-Type: application/json" --data @/home/anna/Downloads/test_procestermijn_nihil.json
+
+def read_xls_row_to_dict(sheet):
```

Comment:
have you considered using [tablib](https://pypi.org/project/tablib/)? It also supports Excel, and has a bit higher-level API.

Reply:
no, I just knew about xlrd and pandas, Thank you! I'll change my code. it'd be much more readable